car-hire-availability.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+
+<!--Just playing around with some HTML on the new car hire optimisations, as practice. Would appreciate some feedback on how I've done here!-->
+
+<head>
+<title>Car Hire | Holiday rentals from £0.70 per day</title>
+</head>
+
+<body>
+<!--Not sure where the background image is added to, but I'm sure I'll work it out!-->
+<!--Is the nav bar actually a span rather than a div? One to double check-->
+  <div class="nav">
+    <div>Logo</div>
+    <div>Contact Us</div>
+  </div>
+
+  <div class="itinerary"></div>
```

Comment:
It was a lapse in concentration - I started with setting out the big divs then went back to fill in the smaller ones and forgot to move the closing tags to wrap them. Have changed it now.